sql/core/src/test/resources/sql-tests/inputs/ansi/interval.sql

```diff
@@ -1,17 +1,6 @@
+--SET spark.sql.parser.optionalIntervalPrefix=false
```

Comment:
do we still need this file?

Reply:
I just checked if we could turn off the optional prefix with ansi enabled. I personally think keeping this is better because the parser logic is a bit complicated for the combination about ansi + optionalIntervalPrefix...